Sources/App/Core/Constants.swift

```diff
@@ -34,5 +34,7 @@ enum Constants {
     static let searchLimit = 20
     static let searchLimitLeeway = 5
 
+    // build system settings
     static let trimBuildsGracePeriod = 4  // hours
+    static let branchBuildDeadTime = 24 // hours
```

Comment:
I've made this a `Constant` instead of an env variable for now. It's just simpler for now and we can always transition it if we find we need more control.